apps/infrastructure/services/uuid_impl.py

```
@@ -0,0 +1,9 @@
+import random
+from apps.usecase.ports.uuid_gen import UUIDGen
+from datetime import datetime, timezone
+
+class UUIDGenImpl(UUIDGen):
+    def new_order_id(self) -> str:
+        today = datetime.now(timezone.utc).strftime("%Y%m%d")
+        seq = random.randint(100000, 999999)
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)